__tests__/app.test.js

```diff
@@ -176,3 +176,77 @@ describe("6.GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe.only("7.POST/api/articles/:article_id/comments", () => {
+  test("responds with statusCode 201 and responds with the posted comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "hjhgfhjf",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject({
+          comment_id: 19,
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: "butter_bridge",
+          body: "hjhgfhjf",
+        });
+      });
+  });
+  test("responds with statusCode 400 if the user sends no body", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No Body");
+      });
+  });
+  test("responds with statusCode 400 if the user sends non-valid username", () => {
```

Comment:
This test should be a 404. The username is in the correct string format but it is not found in the database. Well done for including it though! 👍